tests/test_predicates.py

```diff
@@ -97,6 +97,19 @@ def test_disjoint():
     np.testing.assert_array_equal(actual, expected)
 
     # two scalars
-    a2 = spherely.Point(50, 9)
-    b2 = spherely.LineString([(50, 8), (60, 8)])
+    a2 = spherely.point(50, 9)
+    b2 = spherely.linestring([(50, 8), (60, 8)])
     assert spherely.disjoint(a2, b2)
+
+
+def test_predicates_polygon():
```

Comment:
Move above to below `test_contains` as `test_contains_polygon` ? (given this is mostly testing `contains`)

(I suppose the reason those tests are using points/linestrings is because polygon creation was not yet added when writing those tests?)